src/api-reference/request/v4.endpoints.travel-agency-resources.markdown

```diff
@@ -0,0 +1,77 @@
+---
+title: Travel Request v4 - Travel Agency Resources
+layout: reference
+---
+
+{% include prerelease.html %}
+
+* [Travel Agency Resources](#travel-agency-resource)
+  * [Get the description of a travel Agency](#get-travel-agency-description-resource)
+
+## HTTP Status Codes
```

Comment:
Move this content into the individual requests- it goes under the HTTP Response header, as per the doc template.

Reply:
done